app/uk/gov/hmrc/gatekeeper/controllers/EmailsController.scala

```diff
@@ -111,6 +110,13 @@ class EmailsController @Inject() (
     } yield Ok(emailPreferencesSelectApiView(apis.sortBy(_.displayName), selectedApis.sortBy(_.displayName)))
   }
 
+  def addAnotherApiOption(selectOption: String, selectedAPIs: Option[List[String]]): Action[AnyContent] = anyStrideUserAction { implicit request =>
+      selectOption.toInt match {
```

Comment:
This is bad - don't do this.
You pass in a string and then make an unsafe assumption that it contains an int.
Needs fixing.